README.md

```diff
@@ -40,18 +41,20 @@ Learn more about libp2p at [**libp2p.io**](https://libp2p.io) and follow libp2p'
 ## Install
 **Prerequisite**
 - [Nim](https://nim-lang.org/install.html)
+> The currently supported Nim version is 1.6.18.
+
 ```
 nimble install libp2p
 ```
 
 ## Getting Started
 You'll find the nim-libp2p documentation [here](https://status-im.github.io/nim-libp2p/docs/).
 
-**Go Daemon:**
-Please find the installation and usage intructions in [daemonapi.md](examples/go-daemon/daemonapi.md).
+### Go-Libp2p Daemon
+Remember you'll need to build the `go-libp2p-daemon` binary to use `nim-libp2p`.
```

Comment:
I think `go-libp2p-daemon` is only necessary for running the tests, isn't it?

Reply:
Is that so? Why is it needed to run the tests, then?

Reply:
@Menduist told me this once:

> Why do we have testdaemon? Is it necessary?
Currently the only way we have to test pubsub interopability is using the go-daemon, that's why it's still here
Historically, nim-libp2p began has a simple go-daemon wrapper

Reply:
Added a note for that.